src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServer.cs

```diff
@@ -96,7 +104,8 @@ private static ServiceContext CreateServiceContext(IOptions<KestrelServerOptions
                 SystemClock = systemClock,
                 DateHeaderValueManager = dateHeaderValueManager,
                 ConnectionManager = connectionManager,
-                ServerOptions = serverOptions
+                Resources = new ResourceManager(normalConnections, upgradedConnections),
+                ServerOptions = serverOptions,
```

Comment:
Would be tough in the current implementation. This might be a good cleanup when/if we consolidate RejectionConnection back into FrameConnection.